extensions/wikia/ContentReview/ContentReviewHelper.php

```diff
@@ -112,25 +115,48 @@ public static function isStatusAwaiting( $status ) {
 	}
 
 	public function isDiffPageInReviewProcess( $wikiId, $pageId, $diff ) {
+		global $wgRequest;
```

Comment:
Quite a few methods in the helper now access `$wgRequest` and now the one below access `$wgTitle`. It might be worth either injecting the quest context on object creation and making use of that here (i.e. pass `$this->getContext()` from the API controllers to the constructor here, or `RequestContext::getMain()` if you're outside a WikiaController), or make it extend `ContextSource` (see [here](https://github.com/Wikia/app/blob/22bc3d9bf738/extensions/wikia/RestrictSessions/RestrictSessionsHooks.class.php#L12) and [here](https://github.com/Wikia/app/blob/22bc3d9bf738/extensions/wikia/RestrictSessions/RestrictSessionsHooks.class.php#L50) for an example). This would let you avoid the globals and use `$this->getRequest()`, `$this->getTitle()`, etc. and is the best practice where possibly in MW 1.18+.